ext/fetch/23_request.js

```diff
@@ -431,7 +431,8 @@
         return webidl.converters["Request"](V, opts);
       }
     }
-    return webidl.converters["USVString"](V, opts);
+    // Passed to new URL(...) which implictly converts DOMString -> USVString
+    return webidl.converters["DOMString"](V, opts);
```

Comment:
Since the `RequestInfo` type in the fetch WebIDL definitions is `Request | USVString`, you might want to rename the WebIDL type, to avoid mistakes down the line.